source/release-notes/1.9.txt

```diff
@@ -12,16 +12,46 @@ Release Notes for mongosync 1.9
    :depth: 2
    :class: singlecol
 
-.. include:: /includes/in-dev.rst
-
 .. _1.9.0-c2c-release-notes:
 
 This page describes changes and new features introduced in  
 {+c2c-full-product-name+} 1.9 and the {+c2c-full-beta-program+}.
 
+1.9.0 Release 
+-------------
+
+**Upcoming**
+
+Other Notes 
+~~~~~~~~~~~
+
+New Features: 
+
+- ``mongosync`` now supports migrating documents that have an empty timestamp 
+  values, such as Timestamp(0,0) for source clusters using MongoDB version 6.0 
+  or later. 
+
+- Added a ``stdin`` disclaimer to the ``mongosync`` executable regarding 
+  verification. Users must to accept this disclaimer before ``mongosync``  
+  can run.
+
+- Added ``--acceptDisclaimer`` flag for users that have previously read the 
```

Comment:
Should this be documented in the [configuration](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/configuration/) section too?

Reply:
@jtcovan mentioned that this is being documented in #339. If not, would you mind opening up a follow-up ticket, so we can pick that up and address it?

Reply:
Yes, that is already in #339. Staging link at https://deploy-preview-339--docs-cluster-to-cluster-sync.netlify.app/reference/mongosync/#global-options